server/server/src/router_service.rs

```diff
@@ -8,12 +8,13 @@ use lazy_static::lazy_static;
 use lockbook_crypto::pubkey::ECVerifyError;
 use lockbook_models::api::*;
 use lockbook_models::api::{ErrorWrapper, Request, RequestWrapper};
-use log::{error, warn};
 use prometheus::{register_histogram_vec, HistogramVec, TextEncoder};
 use serde::de::DeserializeOwned;
 use serde::Serialize;
 use std::collections::HashMap;
 use std::sync::Arc;
+use tracing::*;
+use tracing::{span, Level};
```

Comment:
I would just chose one style, either `*` or specify, but at a minimum the second line is unnecessary.